README.md

```diff
@@ -117,12 +117,10 @@ npm test
 
```

Comment:
- [ ] As required for the project, your rtyling to be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/), kindly ensure that you have updated it the match the expected result. Right now, your app has one book there is a big space at the top and with more than 2 books the `Header` is heidden.
## Image with one book
![Screenshot from 2023-08-31 18-40-59](https://github.com/AjroudiRida/bookstore/assets/83042742/624eb616-f1ba-401e-a3f8-1428c8802ff9)

## Image with many books
![Screenshot from 2023-08-31 18-40-32](https://github.com/AjroudiRida/bookstore/assets/83042742/b8d8e8a9-71ee-49df-9b0d-f5be34d24d75)